### PR TITLE
Sublime v1 utility functions

### PR DESCRIPTION
**Pull request by aditya26026**
Functions like amountPerPeriod, interestPerSecond, amountLenderPerPeriod and calculateCurrentPeriod

### CODE REVIEW

File: contracts/Pool/Pool.sol [hunk 1]
```diff
@@ -382,16 +382,31 @@ contract Pool is ERC20PresetMinterPauserUpgradeable,IPool {
...
+    function interestPerSecond(uint256 _principle)
+        public
+        view
+        returns (uint256)
+    {
+        uint256 _interest = ((_principle).mul(borrowRate)).div(365 days);
+        return _interest;
+    }
+    
+    function amountLenderPerPeriod(address _lender)
```
Comment (prateekreddy) [inline contracts/Pool/Pool.sol:394]: Can you rename the function ? Intent is not very clear

File: contracts/Pool/Pool.sol [hunk 2]
```diff
@@ -382,16 +382,31 @@ contract Pool is ERC20PresetMinterPauserUpgradeable,IPool {
...
+        returns (uint256)
+    {
+        return (interestPerSecond(balanceOf(_lender)).mul(repaymentInterval));
     }
 
-    function amountLenderPerPeriod(address lender) public view returns(uint256){
-        
+    function calculateCurrentPeriod() public view returns (uint256) {
+        uint256 _currentPeriod =
+            (block.timestamp.sub(loanStartTime)).div(repaymentInterval);
```
Comment (prateekreddy) [inline contracts/Pool/Pool.sol:404]: Give the error message in sub as well, basically "Loan hasn't started yet" or something on those lines

Comment (aditya26026) [reply contracts/Pool/Pool.sol:404]: will do.

File: contracts/Pool/Pool.sol (hunk 1 above)
Comment (aditya26026) [reply contracts/Pool/Pool.sol:394]: will do.

Comment (aditya26026) [reply contracts/Pool/Pool.sol]: I have changed both amountPerPeriod and amountLenderPerPeriod to an overloaded interestPerPeriod

**Pull request merged**